src/config/userSettings.js

```diff
@@ -15,13 +15,15 @@ const userSettings = {
       iconType: iconType.google,
       type: widgetType.number,
       formatter: financeFormatter,
+      heading: 'Rocket Internet Stock', // Only for displaying widget name in settings modal
     },
     {
       id: uuidv4(),
       key: 'weather',
       display: true,
       iconType: iconType.weather,
       type: widgetType.weather,
+      heading: 'Weather for Berlin', // Only for displaying widget name in settings modal
```

Comment:
What if we configure to check a different city? Let's discuss a different solution.